_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -260,8 +265,12 @@ VABufferID& VABuffersHandler::VABuffersNew(mfxU32 id, mfxU32 pool, mfxU32 num)
 
 void VABuffersHandler::VABuffersDestroy()
 {
+    mfxStatus mfxSts;
+
     for (size_t i = 0; i < m_buf.size(); i++)
-        MFX_DESTROY_VABUFFER(m_buf[i], m_vaDisplay);
+    {
+        mfxSts = CheckAndDestroyVAbuffer(m_vaDisplay, m_buf[i]);
```

Comment:
The same as above about checking mfxSts

Reply:
No need in status in destructor. Just log error maybe.

Reply:
Ok, I see that function VABuffersDestroy return "void" - in this case I suggest to remove "mfxStatus mfxSts;" if it's not used

Reply:
as I stated above - better to have status at least in debug. If no good idea how to use it, I suggest something like
```
#if DEBUG
mfxStatus sts = 
#endif 
CheckAndDestroyVAbuffer(m_vaDisplay, m_buf[i]);
```